docker-compose-tests.yml

```diff
@@ -25,7 +25,7 @@ services:
       - gp-finder-travis-tests-node_modules:/code/node_modules
     environment:
       COVERALLS_REPO_TOKEN: ${COVERALLS_REPO_TOKEN}
-      NODE_ENV: development
+      NODE_ENV: ${NODE_ENV-development}
```

Comment:
Does the variable want setting when the value is either unset or empty or just when it is unset?
See docs for more info: 
https://docs.docker.com/compose/compose-file/compose-file-v2/#variable-substitution